source/use-nunjucks-macros/index.html.md.erb

```diff
@@ -0,0 +1,94 @@
+---
+title: Use Nunjucks macros
+weight: 45
+---
+
+# Use Nunjucks macros
+
+You can use Nunjucks macros to generate the HTML for components when you render a page.
+
+You can also change a component's HTML by passing options into the Nunjucks macro.
+
+If you use Nunjucks macros, your project will be easier to update when a new version of GOV.UK Frontend is released.
+
+To use Nunjucks macros:
+
+- you must [install GOV.UK Frontend with Node.js package manager (npm)](/installing-with-npm/)
+- your application and any frameworks that support it must be written in JavaScript
+
+You do not need to follow this guidance to [use Nunjucks macros in the GOV.UK Prototype Kit](https://design-system.service.gov.uk/get-started/prototyping/#using-nunjucks-macros).
+
+## Set up Nunjucks
+
+1. Make sure you installed Nunjucks when you [installed GOV.UK Frontend with npm](/installing-with-npm).
+
+2. Import the Nunjucks package at the top of your configuration file, for example:
+
+    ```javascript
+    const nunjucks = require('nunjucks')
+    ```
+
+3. Tell Nunjucks where to find the GOV.UK Frontend template and components by adding the following to your configuration file:
+
+    ```javascript
+    nunjucks.configure([
+      "node_modules/govuk-frontend/"
+    ], {
+      autoescape: true
+    })
+    ```
+
+    `autoescape: true` makes the output render as text by default, to help prevent cross-site scripting (XSS) attacks.
+
+4. Use the GOV.UK Frontend template by adding the following at the top of your view file:
+
+    ```javascript
+    {% extends "govuk/template.njk" %}
+    ```
+
+5. Go to the [default page template example](https://design-system.service.gov.uk/styles/page-template/#default) on the GOV.UK Design System website, then copy the Nunjucks code into your view file.
+
+## Importing a component
+
+You can go to any component page on the Design System website, then copy the code from the **Nunjucks** tab of any example.
+
+For example, to add the breadcrumbs component to your page, add the following code from the first example on the [breadcrumbs component page](https://design-system.service.gov.uk/components/breadcrumbs/):
+
+```javascript
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
```

Comment:
Are we happy with the risk of this getting out of sync with the first example on the Breadcrumbs page?

Reply:
Hmm good question. @hannalaakso - is it better to ask users to copy the code from the breadcrumbs page? I can't remember why we specifically included the code now.

Reply:
Yes good catch @36degrees, I think we can ask users to just copy it from the Design System.

Reply:
👍 I've updated in 0526b83